src/MoBi.Core/Commands/SetQuantityValueInBuildingBlockCommand.cs

```diff
@@ -26,7 +26,7 @@ protected SetQuantityValueInBuildingBlockCommand(IQuantity quantity, IBuildingBl
       protected override void ExecuteWith(IMoBiContext context)
       {
          base.ExecuteWith(context);
-         _oldValue = _quantity.Value;
```

Comment:
I tried this in V11.3 and _quantity.Value gave an NaN. In V12 an exception is thrown. There is a convenience method called TryGetValue which returns NaN if the formula can't be evaluated, so I think this makes sense.

IDK why this stopped working though at the Quantity Level.

Reply:
This is surprising that the behaviour is different. Can we track ?

Reply:
Can we write a test for this ?

Reply:
test added and merged